readme.md

```diff
@@ -682,23 +682,44 @@ A function that transform [`Response`](#response) into an array of items. This i
 Type: `Function`\
 Default: [`Link` header logic](source/index.ts)
 
-A function that returns an object representing Got options pointing to the next page. If there are no more pages, `false` should be returned.
+The function takes three arguments:
+- `response` (the current response object)
+- `allItems` (an array of the emitted items)
+- `currentItems` (items from the current response).
+
+It should return an object representing Got options pointing to the next page. If there are no more pages, `false` should be returned.
+
+For example, if you want to stop when the response contains less items than expected, you can use something like this:
+
+```js
+(response, allItems, currentItems) => {
```

Comment:
I would prefer the example to be fully runnable, meaning that it includes the import statement and `got(…)`.

Reply:
I just extended the example.